README.md

```diff
@@ -227,6 +227,11 @@ Volunteers have translated many of our Contributing, How-to, and Code of Conduct
 
 You might notice that there are [some missing translations here](docs/README.md#translations) - perhaps you would like to help out by [contributing a translation](docs/CONTRIBUTING.md#help-out-by-contributing-a-translation)?
 
+## Contributors
+
+<a href="https://github.com/EbookFoundation/free-programming-books/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=EbookFoundation/free-programming-books"  alt="free-programming-books Contributors"/>
```

Comment:
Adding your contribution would downplay the efforts of 2,600 others, so it's best not to include it; every contributor has made valuable contributions.

Reply:
I'm not a big fan of this proposal tbh. The README is unfortunately quite bloated and uncomfortable to read already. ^-^'

Adding more images/widgets that don't actually help the reader or provide them with information they would want, would just further increase how much readers have to sift through before finding information relevant for them.

These widgets are great for relatively empty pages to pad them out, but not when we have more than enough content already.